src/pages/[platform]/build-a-backend/auth/connect-your-frontend/manage-user-attributes/index.mdx

```diff
@@ -901,7 +902,7 @@ async function handleDeleteUserAttributes(
 ```
 </InlineFilter>
 
-#### Next Steps 
+## Next Steps 
 
-- [Set up password change and recovery](/[platform]/build-a-backend/auth/manage-users/manage-passwords/)
-- [Set up custom attributes](/[platform]/build-a-backend/auth/modify-resources-with-cdk/#custom-attributes)
+- [Learn how to set up password change and recovery](/[platform]/build-a-backend/auth/manage-users/manage-passwords/)
+- [Learn how to set up custom attributes](/[platform]/build-a-backend/auth/concepts/user-attributes#custom-attributes)
```

Comment:
`/[platform]/build-a-backend/auth/concepts/user-attributes#custom-attributes` doesn't exist for android swift and flutter, will need to use an InlineFilter here to exclude those platforms from seeing this link

Reply:
ahh that is a mistake. Just updated the page to include those platforms in the meta block